### PR TITLE
added https

### PR DESCRIPTION
**Pull request by dioptre**

### CODE REVIEW

**Comment by MikeInnes** [comment]
Thanks for the patch, glad to have this although I think it could be simplified a little (will leave a note).

File: src/server.jl [hunk 1]
```diff
@@ -39,15 +39,26 @@ function ws_handler(app::App)
   return handler
 end
 
-const default_port = 8000
-
-function serve(s::Server, port = default_port)
-  @async @errs run(s, port)
+function serve(s::Server; args...)
```
Comment (MikeInnes) [inline src/server.jl]: This looks like it'd be a lot simpler as `serve(s::Server, port = 8000; ssl = nothing)`

**Comment by cmcaine** [comment]
Closing. Mux has changed enough that if we ever support HTTPS, we will have to write a new PR.

**Comment by cmcaine** [comment]
Lol, I forgot that we already support HTTPS. Check the tests for example use. An even better reason to close!

**Pull request closed without merging**